lib/modules/manager/helm-values/extract.ts

```diff
@@ -22,7 +22,7 @@ function getHelmDep({
   dep.replaceString = tag;
   dep.versioning = dockerVersioning;
   dep.autoReplaceStringTemplate =
-    '{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}';
+    '"{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}"';
```

Comment:
I think we need to detect quotes, as some people enforce single quotes. so renovate update PR's will fail on CI checks

Reply:
if the user uses double quotes so the new version will be wrapped with double quotes and if the user uses the single quotes the new version will be wrapped with single quotes, is this the expected behavior?

Reply:
yes

Reply:
we can probably simply count single and double quotes and use the one which are more, fallback to double quotes

Reply:
@viceice as I see if the YAML file contains a tag and the version wrapped with double quotes the new version will be wrapped with double quotes, and if wrapped with a single quote the new version will be wrapped with a single quote also.
so I'm thinking to add a regex that checks if the version does not contain any quotes if yes Renovate will add double quotes for the new one.